data/datacatalogue3/molgenis.csv

```diff
@@ -184,11 +184,13 @@ Contacts,,resource,ref,1,TRUE,,Resources,,,,,Resource the contact is affiliated
 Contacts,,role,ontology_array,,,CatalogueOntologies,Contribution types,,,,,Type(s) of contribution or role in the resource,,Contributions,contributionType
 Contacts,,first name,,1,TRUE,,,,,,,First name of the contact person,,Contributions,contact.firstName
 Contacts,,last name,,1,TRUE,,,,,,,Last name of the contact person,,Contributions,contact.surname
+Contacts,,consent,bool,,TRUE,,,,,,,"This person gives permission to display their filled out contact details in the catalogue, which is openly accessible"

```

Comment:
I think description should be 'This person has given permission ....'

Reply:
I will revert this change for now and add it later when the other data model changes are merged.